pkgs/applications/audio/sptlrx/default.nix

```diff
@@ -24,7 +24,9 @@ buildGoModule rec {
   meta = with lib; {
     description = "Spotify lyrics in your terminal";
     homepage = "https://github.com/raitonoberu/sptlrx";
+    changelog = "https://github.com/raitonoberu/sptlrx/releases/tag/v${version}";
     license = licenses.mit;
     maintainers = with maintainers; [ MoritzBoehme ];
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
This is the default for buildGoModule, we no longer recommend setting this if the default platform works (#153464).

```suggestion
```